### ISSUE TITLE
Make cannot find MySQL libraries

### ISSUE THREAD

**Issue by JuanjiJG** [wazuh/wazuh#2026]
## Bug report

**OS**
CentOS 7

**Wazuh version**
3.7.0 from sources

**Wazuh API version**
Not applicable

**Install type**
manager

**Install method**
sources

**Log sample**

```
/usr/bin/ld: cannot find -lmysqlclient
collect2: error: ld returned 1 exit status
make[1]: *** [libwazuhext.so] Error 1
make[1]: Leaving directory `/home/vagrant/wazuh-3.7.0/src'
make: *** [server] Error 2
```

Bug description
----------------
When trying to compile Wazuh to enable MySQL database support, I get an error like the one shown above. This happens on CentOS 7 when installing `mysql-community-*` packages from the [official MySQL webpage](https://dev.mysql.com/downloads/repo/yum/).

**The same error happens when using MariaDB.**

To fix this, I just added `$(OSSEC_LDFLAGS)` to [this line](https://github.com/wazuh/wazuh/blob/4de04533964de79887a46e6ccf61f9541d707cc9/src/Makefile#L995) on the Makefile. After applying the change and recompiling the code, everything works fine.

I'll be updating the issue description if I discover the same bug on different OS or when using PostgreSQL.

**Comment by JuanjiJG** [comment]
Just updated the issue's description. I get the same error when trying with `mariadb-*` packages.

**Comment by DaveVG1** [comment]
Also happens with `postgresql-*` database and on Ubuntu16

**Comment by chemamartinez** [comment]
Fixed at #2108.
